cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -428,10 +430,10 @@ instance FromJSONKey TxIn where
 
 parseTxId :: Parsec.Parser TxId
 parseTxId = do
-  str <- Parsec.many1 Parsec.hexDigit Parsec.<?> "transaction id (hexadecimal)"
-  case deserialiseFromRawBytesHex AsTxId (BSC.pack str) of
-    Just addr -> return addr
-    Nothing -> fail $ "Incorrect transaction id format:: " ++ show str
+  str <- some Parsec.hexDigit <?> "transaction id (hexadecimal)"
```

Comment:
Was there a reason to switch `many1` to `some`? The Parsec version has an implementation specialised to parsers.

Reply:
`some` is a less surprising name for the same thing.